lib/ca_testing/patches/selenium_manager.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module CaTesting
+  module Patches
+    class SeleniumManager < Base
+      private
+
+      def description
+        <<~DESCRIPTION
+          This patch fixes an issue with Selenium4 not bubbling up the httpOnly property of a cookie
+          See: https://github.com/SeleniumHQ/selenium/pull/8958 for more details/discussion including this fix
+        DESCRIPTION
+      end
+
+      def perform
+        ::Selenium::WebDriver::Manager.prepend CookieConverter
+      end
+
+      def deprecation_notice_date
+        Time.new(2021, 12, 25)
```

Comment:
Again. This is fixed in the currently selenium main. But won't be accessible to us until we migrate to selenium4 proper.